compiler/noirc_frontend/src/hir/comptime/value.rs

```diff
@@ -148,6 +150,10 @@ impl Value {
                 ExpressionKind::Literal(Literal::Integer((value as u128).into(), false))
             }
             Value::String(value) => ExpressionKind::Literal(Literal::Str(unwrap_rc(value))),
+            // Forat strings are lowered as normal strings since they are already interpolated.
```

Comment:
```suggestion
            // Format strings are lowered as normal strings since they are already interpolated.
```

What if some of the arguments to interpolate couldn't be resolved? Is there no way to return a format string from a comptime context?

Reply:
If some of the captured arguments couldn't be resolved we'll return an error when evaluating the format string. When an error is returned while lowering a value, `HirExpression::Error` is lowered instead.